src/middlewared/middlewared/plugins/jbof/crud.py

```diff
@@ -372,18 +374,21 @@ async def hardwire_host(self, mgmt_ip, shelf_index, schema, verrors):
                 return
 
             connected_shelf_ips = []
-            for node in ('A', 'B'):
-                connected_shelf_ips = await self.hardwire_node(node, shelf_index, shelf_ip_to_mac, connected_shelf_ips)
+            results = await asyncio.gather(*[self.hardwire_node(node, shelf_index, shelf_ip_to_mac) for node in ('A', 'B')])
+            for (node, connected_shelf_ips) in zip(('A', 'B'), results):
                 if not connected_shelf_ips:
                     # Failed to connect any IPs => error
                     verrors.add(schema, f'Unable to communicate with the expansion shelf (node {node})')
                     return
+                self.logger.debug('Configured node %r: %r', node, connected_shelf_ips)
         else:
             connected_shelf_ips = await self.hardwire_node('', shelf_index, shelf_ip_to_mac)
             if not connected_shelf_ips:
                 # Failed to connect any IPs => error
                 verrors.add(schema, 'Unable to communicate with the expansion shelf')
                 return
+            self.logger.debug('Configured node: %r', connected_shelf_ips)
+        self.logger.debug('Configured host')
```

Comment:
I like the messages, but with only 'Configured host' the context might be lost?

Reply:
I agree with @mgrimesix. Seems that log message might be a bit superfluous.